src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,165 @@
+  const [focused, setFocused] = useState({ email: false, password: false });
+  const [userCredentials, setCredentials] = useState({
+    email: "",
+    password: "",
+  });
+  const { isLoading } = useAppSelector((state: RootState) => state.user);
+  const { loginStart, loginSuccess, loginFail } = userActions();
+  const client = useApiClient();
+  const navigate = useNavigate();
+  const handleLogin = async (event: React.FormEvent<HTMLFormElement>) => {
+    const { email, password } = userCredentials;
+    event.preventDefault();
+    const error = validation(email, password);
+    if (Object.keys(error).length > 0) {
+      setErrorMessage(Object.values(error)[0]);
+      return;
+    }
+    setErrorMessage("");
+    loginStart();
+
+    const res: AxiosResponse<User> | void = await client
+      .post("/sessions", {
+        email,
+        password,
+      })
+      .catch((err) => {
+        loginFail(err.message);
+        setErrorMessage(err.message);
+        return;
+      });
+
+    if (res) {
+      loginSuccess(res.data);
+      navigate("/team/users");
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value, name }: { value: string; name: string } = event.target;
+    setCredentials({ ...userCredentials, [name]: value });
+  };
+
+  const handleFocus = (event: React.FocusEvent<HTMLInputElement>) => {
+    setFocused({ ...focused, [event.target.name]: true });
+  };
+  const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    setFocused({ ...focused, [event.target.name]: false });
+  };
+
+  return (
+    <Box sx={styles.mainContainer}>
+      <Box sx={styles.leftRow}>
+        <Box sx={styles.logoContainer}>
+          <img style={styles.image} src={leftDraw} alt="Drawing" />
+        </Box>
+      </Box>
+      <Box sx={styles.middleRow}>
+        <Box sx={styles.logoContainer}>
+          <img style={{ height: 60 }} src={logo} alt="Generic Soft Logo" />
+        </Box>
+
+        <Box sx={styles.formContainer}>
+          <Typography sx={styles.header}>Sign in to our platform</Typography>
+          <form
+            style={{
+              width: "100%",
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+            }}
+            onSubmit={handleLogin}
+          >
+            <Typography
+              sx={{
+                ...styles.helperText,
+                color: focused.email ? THEME_COLORS.purple : "",
+              }}
+            >
+              Your Email
+            </Typography>
+            <CustomInput
+              name="email"
+              onFocus={handleFocus}
+              onBlur={handleBlur}
+              onChange={handleChange}
+              endAdornment={<MailOutlineIcon sx={styles.icons} />}
+            />
+
+            <Typography
+              sx={{
+                ...styles.helperText,
+                color: focused.password ? THEME_COLORS.purple : "",
```

Comment:
Create a variable for this one  - color: focused.password ? THEME_COLORS.purple : "",

Example:  const isPasswordFocusedStyle =  focused.password ? THEME_COLORS.purple : "",

color: isPasswordFocusedStyle